cluster/gce/manifests/kube-scheduler.manifest

```diff
@@ -45,7 +45,8 @@
     "livenessProbe": {
       "httpGet": {
         "host": "127.0.0.1",
-        "port": 10251,
+        "port": 10259,
+        "scheme": "HTTPS",
         "path": "/healthz"
```

Comment:
is there a way to verify that /metrics is working?

Reply:
Do you mean besides the e2e?

Reply:
Yeah, I was wondering if there is an e2e test for the metrics.

Reply:
`pull-kubernetes-e2e-gce-100-performance` relies on the /metrics endpoint to be working

Reply:
can you verify if such job is using the secure port?

Reply:
Maybe we should be doing this change in 2 stages: first PR to move everything to the secure port, and then second to remove the insecure port.

Reply:
Discussed in https://github.com/kubernetes/kubernetes/pull/96216#issuecomment-722435252 and done through https://github.com/kubernetes/kubernetes/pull/96550 and https://github.com/kubernetes/perf-tests/pull/1647.

Reply:
I see. Can you make a PR just for the manifest and any other uses of port 10251.

Reply:
What other uses of 10251 do you have in mind? Can you give me a list of all the locations in questions?

Reply:
Me neither. Though, the same changes were carried in KCM as well. I prefer to have this change here so it's part of the same PR.

Reply:
It's good to be able to revert one PR without reverting the other. And it can give us extra confidence in that removing the insecure serving is possible.

Reply:
I can think only of changes in test/integration/serving/serving_test.go and cluster/gce/manifests/kube-scheduler.manifest in k\k repository. I can create a separate PR for those if you want. There's nothing else left besides https://github.com/kubernetes/kubernetes/blob/master/test/fixtures/doc-yaml/admin/high-availability/kube-scheduler.yaml#L18.

Reply:
yes, one PR for migrating all of those is good.

Reply:
https://github.com/kubernetes/kubernetes/pull/102857 PTAL